content/en/os/1.13.x/update/guidelines.markdown

```diff
@@ -0,0 +1,62 @@
+# Guidelines for Updating In-Place
+
+## Introduction
+
+There are some guidelines and considerations to keep in mind when updating your Bottlerocket nodes _in-place_.
+Some movements between versions are updates, some are not updates, and there are some caveats you may need to consider based on the history of the project.
+
+## Update Paths
+
+Certain movements between releases of Bottlerocket are accessible through an _in-place_ update path.
+
+### Between Minor or Patch Versions of the Same Variant
+
+With the exception of major versions, it is possible to update from one released version of a variant to another released version of the same variant.
+In particular, the following update paths work:
+
+- Minor version to minor version (e.g. `1.10.0` to `1.11.0`)
+- Patch version to patch version (e.g. `1.9.1` to `1.9.2`)
+- Minor version to patch version (e.g. `1.10.0` to `1.10.1`)
+- Patch version to minor version (e.g. `1.10.1` to `1.11.0`)
+
+Furthermore, skipping minor or patch versions of the same variant is allowed.
+There is a [caveat discussed below](#too-many-releases-between-versions), however, if there are too many minor or patch releases between your initial and target version.
+For example, a possible update path is `aws-k8s-1.23` version `1.10.1` to `aws-k8s-1.23` version `1.12.0`.
+Bottlerocket runs all update [settings migrations](https://github.com/bottlerocket-os/bottlerocket/tree/develop/sources/api/migration#data-store-migration) between the initial and target versions in sequence if a version is skipped.
+So, for the `1.10.1` to `1.12.0` example, the [settings migrations](https://github.com/bottlerocket-os/bottlerocket/tree/develop/sources/api/migration#data-store-migration) from `1.11.0` and `1.11.1` are run as part of the update to `1.12.0`.
+
+## Limits
+
+### Caveats
+
+You may need to consider the following caveats when updating your Bottlerocket nodes.
```

Comment:
do you have other caveats planned? If not, eliminate line 30 and just flatten the limits section to one level (e.g. 'caveats and limits')

Reply:
I wanted to keep the structure open to potential future caveats. However, since there aren't any now, I'll flatten the section as suggested